src/Polynomials.jl

```diff
@@ -372,7 +372,7 @@ function polyval{T,S}(p::Poly{T}, x::S)
     if lenp == 0
         return zero(R) * x
     else
-        y = convert(R, p[end]) + zero(T)*x
+        y = convert(R, p[end])
```

Comment:
I don't understand why `zero(T)*x` is here. If there is any reason, perhaps changing to `zero(T)*zero(x)`?